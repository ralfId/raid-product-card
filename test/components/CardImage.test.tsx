import React from 'react';
import renderer from 'react-test-renderer';
import { CardImage, ShoppingCard } from '../../src/components';
import { product2 } from '../data/data';


describe('card image', () => {
    test('debe mostrar la imagen', () => {
        const wrapper = renderer.create(<CardImage image='../ruta-de-img' />);

        //console.log(wrapper.toJSON());
        expect(wrapper.toJSON()).toMatchSnapshot();
    })
})

describe('card image dentro de ShoppingCard', () => {
    test('debe mostrar la imagen', () => {
        const wrapper = renderer.create(
        <ShoppingCard product={product2}>
        {
            () => (

                <CardImage  />

            )
        }
    </ShoppingCard>
        );

        //console.log(wrapper.toJSON());
        expect(wrapper.toJSON()).toMatchSnapshot();
    })
})