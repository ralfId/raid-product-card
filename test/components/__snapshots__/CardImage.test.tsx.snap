// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`card image debe mostrar la imagen 1`] = `
<img
  alt="coffee mug"
  className="productImg undefined"
  src="../ruta-de-img"
/>
`;

exports[`card image dentro de ShoppingCard debe mostrar la imagen 1`] = `
<div
  className="productCard undefined"
>
  <img
    alt="coffee mug"
    className="productImg undefined"
    src="./coffee-mug.png"
  />
</div>
`;
